docs-md/web/utilities.md

```diff
@@ -0,0 +1,84 @@
+---
+title: JavaScript Utilities
+description: Capacitor's JavaScript Utilities
+url: /docs/web/utilities
+contributors:
+  - dotNetkow
+---
+
+# JavaScript Utilities
```

Comment:
Am I missing any? [Here's the list](https://github.com/ionic-team/capacitor/blob/master/core/src/definitions.ts#L69). The others appear to be exclusively for use by Capacitor itself.